rustls-libssl/tests/config.c

```diff
@@ -148,6 +150,86 @@ void test_min_max_versions(void) {
   SSL_free(ssl);
 }
 
+static const char *verify_modes[] = {
+    NULL,   "",        "Ludicrous", "Ludicrous,Absurd",
+    "Peer", "Request", "Require",   "Request,Require",
+};
```

Comment:
The unsupported verify modes (Once, etc) are omitted here because OpenSSL recognizes them and we don't so the verify mode on each side wouldn't match up.